templates/telemetry_derived/main_summary_v4.sql

```diff
@@ -0,0 +1,348 @@
+SELECT
+  document_id,
+  client_id,
+  sample_id,
+  metadata.uri.app_update_channel AS channel,
+  normalized_channel,
+  normalized_os_version,
+  metadata.geo.country,
+  metadata.geo.city,
+  metadata.geo.subdivision1 AS geo_subdivision1,
+  metadata.geo.subdivision2 AS geo_subdivision2,
+  environment.system.os.name AS os,
+  JSON_EXTRACT_SCALAR(additional_properties, "$.environment.system.os.version") AS os_version,
```

Comment:
@acmiyaguchi is this supposed to be missing from the main ping schema? see also every case of `JSON_EXTRACT_SCALAR` in this query

Reply:
Yes, it's typed as [either a string or an integer](https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/520887b3cf6e428aec87840fa83513a5984162a6/schemas/telemetry/main/main.4.schema.json#L956-L961), so it's dropped from the table schema.

Reply:
ah, thanks for the clarification ^_^